packages/extension-chakra-storefront/config/default.json

```diff
@@ -1,50 +1,144 @@
 {
     "enabled": true,
     "activeDataEnabled": false,
+    "maxCacheAge": 900,
+    "staleWhileRevalidate": 900,
     "commerceAPI": {
         "proxyPath": "/mobify/proxy/api",
         "parameters": {
-            "clientId": "<CLIENT_ID>",
-            "organizationId": "<ORGANIZATION_ID>",
-            "shortCode": "<SHORT_CODE>",
-            "siteId": "<SITE_ID>"
+            "clientId": "c9c45bfd-0ed3-4aa2-9971-40f88962b836",
+            "organizationId": "f_ecom_zzrf_001",
+            "shortCode": "8o7m175y",
+            "siteId": "RefArchGlobal"
         }
     },
-    "defaultSite": "<SITE_ID>",
+    "defaultSite": "RefArchGlobal",
+    "defaultLocale": "en-US",
+    "defaultSiteTitle": "Retail React App",
     "einsteinAPI": {
         "host": "https://api.cquotient.com",
-        "einsteinId": "<EINSTEIN_ID>",
-        "siteId": "<EINSTEIN_SITE_ID>",
+        "einsteinId": "1ea06c6e-c936-4324-bcf0-fada93f83bb1",
+        "siteId": "aaij-MobileFirst",
         "isProduction": false
     },
+    "search": {
+        "recentSearchLimit": 5,
+        "recentSearchKey": "recent-search-key",
+        "recentSearchMinLength": 3,
+        "defaultSearchParams": {"limit": 25, "offset": 0, "sort": "best-matches", "refine": []},
+        "defaultLimitValues": [25, 50, 100]
+    },
     "pages": {
-        "Account": "/account",
-        "Cart": "/cart",
-        "Checkout": "/checkout",
-        "CheckoutConfirmation": "/checkout/confirmation/:orderNo",
-        "Home": "/",
-        "Login": "/login",
-        "Registration": "/registration",
-        "ResetPassword": "/reset-password",
-        "LoginRedirect": "/callback",
-        "ProductDetail": "/product/:productId",
-        "ProductList": ["/search", "/category/:categoryId"]
+        "Account": {
+            "path": "/account"
+        },
+        "Cart": {
+            "path": "/cart"
+        },
+        "Checkout": {
+            "path": "/checkout"
+        },
+        "CheckoutConfirmation": {
+            "path": "/checkout/confirmation/:orderNo"
+        },
+        "Home": {
+            "path": "/",
+            "productLimit": 10,
+            "mainCategory": "newarrivals"
+        },
```

Comment:
I think I like the current structure. If there are specific config for pages, they go in here. Otherwise, if there are more global configs, they go to the root (like `search` for example).